### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by xwikiorgci**
Translations update from [Weblate](https://l10n.xwiki.org/projects/xwiki-platform/xwiki-core-resources/) for XWiki Platform/XWiki Core Resources.

Current translation status:

Weblate translation status

If a new locale is introduced by this pull request make sure to follow https://dev.xwiki.org/xwiki/bin/view/Community/L10N/SupportedLocales/#HNewlocalechecklist before applying it.

### CODE REVIEW

**Review by surli — CHANGES_REQUESTED** [review]
Some things to fix before merging.

**Comment by surli** [comment]
Interesting: we should review our translation files, it looks like `#@deprecatedstart` was transformed to `@deprecatedstart` so it's considered as a translation somehow and we're getting missing translations key for it in the diff.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties
Comment (surli) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:2442]: This is not good in the source property file.

Comment (tmortagne) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:2442]: And not only that, weblate cleaned a lot of empty lines, which is not its job. It's like it tried to regenerate this file as a translation instead of the source file.

**Comment by tmortagne** [comment]
> Interesting: we should review our translation files, it looks like #@deprecatedstart was transformed to @deprecatedstart so it's considered as a translation somehow and we're getting missing translations key for it in the diff.

I fixed it in the source, but not sure how to tell Weblate to re-regenerate the translation files so that it remove the change on this "key".

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties
Comment (surli) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:2442]: > And not only that, weblate cleaned a lot of empty lines, which is not its job. It's like it tried to regenerate this file as a translation instead of the source file.

So you have to be aware that there's no dedicated mechanism for source file in weblate code right now: we always re-generate the entire file from the list of translations. I haven't provided a specific implem for source files. So those changes makes sense from that point of view.

**Pull request merged**
